bot/cogs/match.py

```diff
@@ -224,6 +224,7 @@ class MatchCog(commands.Cog):
     def __init__(self, bot):
         """ Set attributes. """
         self.bot = bot
+        self.pending_task = None
 
     async def draft_teams(self, message, users):
```

Comment:
Need to add a `guild` argument to this method, so we know what guild the team draft is for.

Reply:
you meant something like this?

Reply:
Yeah, the way it displayed the code made me think the changes were inside of the draft_teams function. Disregard.